tools/api-key-rotation/tests/test_compare_dates.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2021 Google LLC.
+# This software is provided as is, without warranty or representation for any use or purpose. # pylint: disable = line-too-long
+# Your use of it is subject to your agreement with Google.
+#
```

Comment:
Please use the Apache license copyright header

Reply:
Updated